website/docs: clarify binding documentation (#23482)
goauthentik/authentik: website/docs/add-secure-apps/bindings-overview/index.md

---
title: Bindings in authentik
---

A binding connects one authentik object to another object that uses it. Bindings let authentik answer two common questions:

- Where should authentik evaluate this policy, user, or group?
- Where should authentik insert this stage into a flow?

A policy answers "should this request pass?" A policy binding decides where authentik asks that question.

> For step-by-step links, see [Work with bindings](./work-with-bindings.md).

Reviewer: Links?

## Binding types

> The two binding types that you will work with most often are policy bindings and flow-stage bindings.

Reviewer: ```suggestion
The two most common binding types are policy bindings and flow-stage bindings.
```

- **Policy bindings** attach a policy, user, or group to a target that supports policy binding.
- **Flow-stage bindings** attach a stage to a flow and define where that stage runs.

Use a policy binding when you want to control whether a target is allowed, denied, or shown. Use a flow-stage binding when you want to place a stage into a flow and decide when it should run.

## Where bindings are used

The policy binding system is shared by several authentik objects. You can bind policies, users, and groups to these targets:

> | Target                  | What the binding controls                               |

Reviewer: I think that the "What the binding controls" column needs work. 

The description for "Flow-stage binding in that flow" is "Whether a specific stage runs in that flow.".

I think that a clearer explanation given the lead in sentence of "You can bind policies, users, and groups to these targets:"

Is: "Whether a specific stage applies/is shown to a user running the flow" or something to the same effect.

OR we need to change the lead in sentence to be broader. Or a mix of both changes.

---
title: Bindings in authentik
sidebar_position: 4
sidebar_label: "Bindings"
---

A binding connects one authentik object to another object that uses it. Bindings let authentik answer two common questions:

- Where should authentik evaluate this [policy](../../customize/policies/index.md), user, or group?
- Where should authentik insert this [stage](../flows-stages/stages/index.md) into a flow?

A policy answers "should this request pass?" A policy binding decides where authentik asks that question.

> For step-by-step links, see [Work with bindings](./work-with-bindings.md).

Reviewer: ```suggestion
For a step-by-step guide, see [Work with bindings](./work-with-bindings.md).
```
